indico/modules/bootstrap/controllers.py

```diff
@@ -51,7 +52,8 @@ def _process_GET(self):
                                operating_system=get_os(),
                                postgres_version=get_postgres_version(),
                                indico_version=indico.__version__,
-                               python_version=python_version())
+                               python_version=python_version(),
+                               show_local_warning=(is_private_url(request.url_root) or config.DEBUG))
```

Comment:
I'd reverse the order of the checks, reading `config.DEBUG` is cheaper than a call that involves network activity